src/main.rs

```diff
@@ -84,6 +80,16 @@ enum Commands {
         /// arguments. You can only supply version numbers or a custom branch, not both.
         #[arg(long, verbatim_doc_comment)]
         branch: Option<String>,
+        /// The type of deployment.
+        ///
+        /// Possible values are 'development', 'production' or 'staging'. The value used will
+        /// determine the sizes of VMs, the number of VMs, and the number of nodes deployed on
+        /// them. The specification will increase in size from development, to staging, to
+        /// production.
+        ///
+        /// The default is 'development'.
+        #[clap(long, default_value_t = EnvironmentType::Development, value_parser = parse_deployment_type, verbatim_doc_comment)]
+        environment_type: EnvironmentType,
```

Comment:
The docs for the other arguments read:
```
        /// If the argument is not used, the value will be determined by the 'deployment-type'
        /// argument.
```

So I think either this argument has to be renamed OR the docs have to be updated everywhere.

Reply:
Ah, thanks. The argument and enum was initially `DeploymentType`, but I changed my mind.

I've now updated those.